test/health_cards/covid_health_card_test.rb

```diff
@@ -24,6 +24,7 @@ class COVIDHealthCardTest < ActiveSupport::TestCase
     assert_includes type, HealthCards::CardTypes::VC_TYPE[0]
     assert_includes type, 'https://smarthealth.cards#covid19'
     assert_includes type, 'https://smarthealth.cards#immunization'
+    assert_includes type, 'https://smarthealth.cards#labresult'
```

Comment:
Another way to potentially increase code coverage and robustness would be to add a lab result Observation resource to the covid_bundle tested in this file, and add asserts to check for the lab result entries every time it is similarly done for the immunizations.

Reply:
resolved